src/wallet/wallet.cpp

```diff
@@ -2501,10 +2502,11 @@ bool CWallet::CommitTransaction(CWalletTx& wtxNew, CReserveKey& reservekey, CCon
 
         if (fBroadcastTransactions)
         {
+            CValidationState state;
             // Broadcast
-            if (!wtxNew.AcceptToMemoryPool(maxTxFee)) {
+            if (!wtxNew.AcceptToMemoryPool(maxTxFee, state)) {
                 // This must not fail. The transaction has already been signed and recorded.
```

Comment:
IIRC the trouble means that you end up with transactions in your wallet that may never reach the mempool or a block.

But consider my nit a µnit, it can be done in a later pull.